postfix/goss/with-sasl_passwd/goss.yaml

```diff
@@ -11,5 +11,5 @@ port:
       ip:
         - 0.0.0.0
 file:
-  /etc/postfix/sasl_passwd.db:
+  /etc/postfix/sasl_passwd.lmdb:
```

Comment:
@cw-sakamoto 
Please tell me. Is it in the form of a new postfix with a different extension?

Reply:
@cw-ozaki 

I think that's probably true, I checked locally with base image set to `alpine-sdk:3.11` (postfix 3.4) and it was still .db.
I looked for detailed documentation, but couldn't find any.